space-age/example.java

```diff
@@ -0,0 +1,69 @@
+public class SpaceAge {
+
+    private enum Planet {
+        EARTH, MERCURY, VENUS, MARS, JUPITER, SATURN, URANUS, NEPTUNE
+    }
+
+    private double seconds;
+
+    private static final double EARTH_ORBITAL_PERIOD_IN_SECONDS = 31557600.0;
+    private static final int PRECISION = 2;
+    private static final Map<Planet, Double> relativeOrbitalPeriods = new HashMap<>();
+
+    static {
```

Comment:
Could fold this into the `Planet` enum itself. (Oddly enough, it's in the official tutorial :-)

https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html